src/components/data-management/SamplesTableCells.jsx

```diff
@@ -127,25 +127,23 @@ const UploadCell = (props) => {
     }
   };
   const onDownload = () => {
-    downloadSampleFile(activeExperimentId, sampleUuid, uploadDetailsModalData.name, selectedTech);
+    downloadSampleFile(
+      activeExperimentId, sampleUuid, uploadDetailsModalData.fileCategory,
```

Comment:
Do we even need the  fileCategory? Isnt experimentId and sampleUuid enough information for the download?

Reply:
No, because there can be more than one sample file, you need to know what file of the sample you are choosing